src/components/Calculator.js

```diff
@@ -1,46 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
+import buttonNames from './BtnNames';
 import '../App.css';
 
 // eslint-disable-next-line react/prefer-stateless-function
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-    this.handleClick = this.handleClick.bind(this);
-  }
+function Calculator() {
```

Comment:
- [ ] _[OPTIONAL BEFORE NEXT MILESTONE]_ Kindly, omit the usage of the function keyword, instead, use the arrow function declaration.

Reply:
Hi @omar-labana, 

Thanks for the feedback, I will change it now and use the arrow function declaration.